src/ListItem.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export default function ListItem({ destination, notes }) {
```

Comment:
"ListItem" does not really fit semantically to a section with an `<h2>`and `<p>`, does it?

Reply:
Yeah, I thought about that. Cannot think of a good name as the whole list is a "Diving Wishlist" and each section is an item on that list. I didn't want to name it "dive destination" as that is basically the name of the input on the form.... :-/